Remotion/Web/Core/UI/Controls/WebTreeNodeCollection.cs

```diff
@@ -23,6 +25,18 @@ namespace Remotion.Web.UI.Controls
   /// <summary> A collection of <see cref="WebTreeNode"/> objects. </summary>
   public class WebTreeNodeCollection : ControlItemCollection
   {
+    /// <summary>
+    /// Groups the top level <see cref="WebTreeNode"/>s by their <see cref="WebTreeNode.Category"/>
+    /// </summary>
+    /// <param name="webTreeNodes">The nodes of the <see cref="WebTreeView"/></param>
+    /// <returns></returns>
```

Comment:
You can either remove the the `return` or add a meaningful description. Please don't leave it empty.